.schema/config.schema.json

```diff
@@ -201,6 +201,37 @@
       "examples": [
         "1h"
       ]
+    },
+    "grantJWT": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Authorization Grants using JWT configuration",
+      "properties": {
+        "client_auth_optional": {
+          "type": "boolean",
+          "description": "If false, client authentication is required to get access token.",
+          "default": false
+        },
+        "id_optional": {
+          "type": "boolean",
+          "description": "If false, JTI claim must be present in JWT assertion.",
+          "default": false
+        },
+        "issued_date_optional": {
+          "type": "boolean",
+          "description": "If false, IAT claim must be present in JWT assertion.",
+          "default": false
+        },
+        "max_duration": {
+          "description": "Configures how long JWT assertion is considered valid, since being issued.",
```

Comment:
```suggestion
          "description": "Configures what the maximum age of a JWT assertion can be. Overrides the JWT's EXP claim. Useful as a safety measure and recommended to not be set to 720h max.",
```

Reply:
Well it doesn't overrides EXP claim, it compares EXP claim to current setting. Made some corrections in description.